test/success/num.prl

```diff
@@ -86,7 +86,7 @@ Thm NatSymm : [
    (path {_} nat a b)
    (path {_} nat b a))
 ] by [
-  lam a. lam b. lam pab.
+  lam a b pab.
```

Comment:
Can we have something called "intro(s)" which basically does the same thing but with a warning that this is RedPRL not Coq/...?